web.py

```diff
@@ -330,10 +336,10 @@ async def pair(base: str, quote: str, session=Depends(get_db)):
         "quote_symbol": quote.symbol,
         "last_block": last_swap.block,
         "last_txid": last_swap.hash,
-        "last_price": last_price,
-        "base_volume": base_volume,
-        "quote_volume": quote_volume,
-    }
+        "last_price": FormattedFloat(last_price),
+        "base_volume": FormattedFloat(base_volume) or 0,
+        "quote_volume": FormattedFloat(quote_volume) or 0,
```

Comment:
Are you sure you didn't want `or 0` inside?

Reply:
Yes. If I move it inside, the result is `0.` and `0` with current approach